index.html

```diff
@@ -0,0 +1,215 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="stylesheet" type="text/css" href="style-reset.css">
+  <link rel="stylesheet" type="text/css" href="style.css">
+  <link rel="stylesheet" href="//use.fontawesome.com/releases/v5.0.7/css/all.css" crossorigin="anonymous">
+  <link rel="preconnect" href="https://fonts.gstatic.com">
+  <link href="https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,400;1,900&display=swap" rel="stylesheet">
+  <title>Smashingmagazine Clone</title>
+</head>
+
+<body>
+  <nav>
+    <div class="nav-logo">
+      <a href="#"><span>Image</span></a>
+    </div>
+    <div class="nav-links">
+      <a href="#">Link</a>
+      <a href="#">Link</a>
+      <a href="#">Link</a>
+      <a href="#">Link</a>
+      <a href="#">Link</a>
+      <a class="colored" href="#">Link</a>
+    </div>
```

Comment:
- [ ] The links should be inside a `<ul>` tag and the links wrapped around `<li>` tags

Reply:
Thanks for the suggestions, made the changes accordingly.